src/main/java/classify/user/InputParsing.java

```diff
@@ -106,6 +108,11 @@ public static void parseUserCommand(String[] userCommand, ArrayList<Student> mas
                 parseListCommand(masterStudentList, archiveList, recentlyDeletedList, in);
             }
             break;
+        
+        case PROCESS_FILE:
```

Comment:
Valid naming convention for magic literals, standardised with the rest of our programme 👍